Elmish.WPF.sln

```diff
@@ -3,56 +3,51 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.28803.352
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".paket", ".paket", "{021C7D6F-30F1-41CC-B8D6-3460EFF300C3}"
-	ProjectSection(SolutionItems) = preProject
-		..\paket.dependencies = ..\paket.dependencies
-	EndProjectSection
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Elmish.WPF", "src\Elmish.WPF\Elmish.WPF.fsproj", "{8C6E8D34-7205-4C57-9722-87E30E4FC5CE}"
```

Comment:
My personal preference is to have the solution file in the `src` directory.

Is there an advantage to moving it above `src` and into the root of the repository?

Reply:
The advantage is that `dotnet build` works from the repo root without having to specify a sub-directory. So it somewhat simplifies CI and CLI build. Other than that I'm comfortable with having it in `src`. I have no problems moving it back if you prefer that.

Reply:
The reason I like it in `src` is because I think of it has part of the code in contrast to "DevOps-y" things (like YAML files) that I put in the repository root or in other directories.  In my experience, the technical advantages (such as being able to omit certain build parameters as you mentioned) are small compared to what I think is a somewhat larger conceptual advantage.

So, yea, I would prefer it to remain in `src` if you don't mind.